mathesar/api/db/permissions/database.py

```diff
@@ -0,0 +1,16 @@
+from django.db.models import Q
+from rest_access_policy import AccessPolicy
+
+from mathesar.models.users import Role
+
+
+class DatabaseAccessPolicy(AccessPolicy):
+
+    @classmethod
+    def scope_queryset(cls, request, qs):
+        if not request.user.is_superuser:
+            allowed_roles = (Role.MANAGER.value,)
+            if request.method.lower() == 'get':
+                allowed_roles = allowed_roles + (Role.EDITOR.value, Role.VIEWER.value)
```

Comment:
Why would permissions be different for get? This seems like a bad idea.

Reply:
Or rather, it seems like the roles are set up incorrectly.

Reply:
Anyone with an `Editor` or `Viewer` role can only view the related items, Create access is allowed for `Manager` only. I am curious to know why this is a bad idea.

Reply:
After talking synchronously with @silentninja , I'm satisfied that this makes sense.